hoot-services/src/main/java/hoot/services/controllers/grail/RunDiffCommand.java

```diff
@@ -0,0 +1,78 @@
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. DigitalGlobe
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2016, 2017, 2018, 2019 DigitalGlobe (http://www.digitalglobe.com/)
+ */
+package hoot.services.controllers.grail;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import hoot.services.command.CommandResult;
+import hoot.services.command.ExternalCommand;
+
+
+class RunDiffCommand extends GrailCommand {
+    private static final Logger logger = LoggerFactory.getLogger(RunDiffCommand.class);
+
+    RunDiffCommand(String jobId, GrailParams params, String debugLevel, Class<?> caller) {
+        super(jobId,params);
+
+        logger.info("Params: " + params);
+
+        // These are all of the non-default options from the UI
+        List<String> options = new LinkedList<>();
+        options.add("convert.ops=hoot::DecomposeBuildingRelationsVisitor");
+        options.add("element.cache.size.node=10000000");
+        options.add("element.cache.size.relation=2000000");
+        options.add("element.cache.size.way=2000000");
+        options.add("map.cleaner.transforms=hoot::ReprojectToPlanarOp;hoot::DuplicateWayRemover;hoot::SuperfluousWayRemover;hoot::IntersectionSplitter;hoot::UnlikelyIntersectionRemover;hoot::DualWaySplitter;hoot::ImpliedDividedMarker;hoot::DuplicateNameRemover;hoot::SmallWayMerger;hoot::RemoveEmptyAreasVisitor;hoot::RemoveDuplicateAreaVisitor;hoot::NoInformationElementRemover;hoot::CornerSplitter;hoot::RubberSheet");
+        // options.add("match.creators=hoot::NetworkMatchCreator;hoot::ScriptMatchCreator,Area.js;hoot::BuildingMatchCreator;hoot::ScriptMatchCreator,PoiGeneric.js;hoot::PoiPolygonMatchCreator;hoot::ScriptMatchCreator,PowerLine.js;hoot::ScriptMatchCreator,Railway.js;hoot::ScriptMatchCreator,LinearWaterway.js");
+        // options.add("merger.creators=hoot::NetworkMergerCreator;hoot::ScriptMergerCreator;hoot::BuildingMergerCreator;hoot::PoiPolygonMergerCreator");
+        options.add("match.creators=hoot::HighwayMatchCreator;hoot::ScriptMatchCreator,Area.js;hoot::BuildingMatchCreator;hoot::ScriptMatchCreator,PoiGeneric.js;hoot::PoiPolygonMatchCreator;hoot::ScriptMatchCreator,PowerLine.js;hoot::ScriptMatchCreator,Railway.js;hoot::ScriptMatchCreator,LinearWaterway.js");
+        options.add("merger.creators=hoot::HighwaySnapMergerCreator;hoot::ScriptMergerCreator;hoot::BuildingMergerCreator;hoot::PoiPolygonMergerCreator");
+        options.add("reader.conflate.use.data.source.ids.1=true");
+        options.add("reader.conflate.use.data.source.ids.2=false");
+        options.add("way.subline.matcher=hoot::MaximalSublineMatcher");
+        options.add("conflate.pre.ops=hoot::CornerSplitter;");
```

Comment:
@maxgrossman I think we want to use conf files here instead of discrete options.